unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/CharacterPreview/CharacterPreviewController.cs

```diff
@@ -46,14 +47,19 @@ public enum CameraFocus
         [SerializeField] private Transform baseAvatarContainer;
         [SerializeField] private BaseAvatarReferences baseAvatarReferencesPrefab;
 
+        [SerializeField] private GameObject avatarShadow;
+        [SerializeField] private Light avatarLight;
+
         private Service<IAvatarFactory> avatarFactory;
 
         private IAvatar avatar;
         private readonly AvatarModel currentAvatarModel = new () { wearables = new List<string>() };
         private CancellationTokenSource loadingCts = new ();
-
         private IAnimator animator;
         private Quaternion avatarContainerDefaultRotation;
+        private Transform cameraTransform;
+        private (float minZ, float maxZ, float minY, float maxY, float minX, float maxX) cameraLimits;
```

Comment:
You can use Bounds here

Reply:
I had never used the `Bounds` data type! It has been very useful, thanks for the tip! Done here: https://github.com/decentraland/unity-renderer/pull/5162/commits/aedad45cd33b4edb672c3c41ac6d555a365eed5b